document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/FunctionOperator.java

```diff
@@ -4,6 +4,7 @@
   // Unary operations
   ABS,
   FLOOR,
+  LENGTH,
```

Comment:
I hope, all these operations are in need, right?

Reply:
As of now, we use only LENGTH. 

However, would like to keep minimalistic operations here to avoid back-and-forth library changes.